goal_src/goal-lib.gc

```diff
@@ -438,16 +438,23 @@
      )
   )
 
-(defmacro swhen (condition &rest body)
-  "Same as when, but saves the branch condition onto a variable named bc"
+(defmacro aif (condition true false)
+  "Anaphoric if, similar to Common Lisp"
 
-  `(let ((bc ,condition))
-      (if bc
-          (begin ,@body)
+  `(let ((it ,condition))
+      (if it
+          ,true
+          ,false
           )
       )
   )
 
+(defmacro awhen (condition &rest body)
+  "Anaphoric when, similar to Common Lisp"
```

Comment:
thanks dictionary, this makes it more clear
![image](https://user-images.githubusercontent.com/48171810/163647886-a2e50f56-5c09-45f9-a822-7582831fff89.png)

Reply:
Lol, this is what I found first http://community.schemewiki.org/?anaphoric-if

Reply:
(I don't think "anaphoric when" is a lisp thing, only "anaphoric if", so I'm kind of cheating)